client/state/plans/selectors/test/index.js

```diff
@@ -0,0 +1,106 @@
+/**
+ * External dependencies
+ */
+import { expect } from 'chai';
```

Comment:
Nitpick: we are using Jest as our primary unit testing framework now, so it's preferable to not use `chai` here :)
e.g. Here is a randomly picked example: https://github.com/Automattic/wp-calypso/blob/master/client/state/data-layer/wpcom/me/block/sites/delete/test/index.js